app/controllers/orders_controller.rb

```diff
@@ -11,27 +11,28 @@ def new
 
   # POST /orders
   def create
-    @user = User.first
-    @order = Order.create!( customer_id: @user.id, state: "cart" )
+    @order = Order.create!( customer_id: current_user.id, state: "cart" )
     if @order.valid?
       flash[:notice] = 'successfull Created'
       redirect_to order_path(@order)
     else
       flash[:notice] = @order.errors.full_messages
       render :new  
     end
+    binding.pry
```

Comment:
Cycle 0 - Not done properly. Make PR again after cycle 0
@fahad694 
Closing PR